lib/components/Selection control/Checkbox/index.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+    ComponentProps,
+    forwardRef,
+    ReactNode,
+    useState,
+    useEffect,
+} from "react";
+import clsx from "clsx";
+import "./checkbox.scss";
+import {
+    StandaloneSquareCheckFillIcon,
+    StandaloneSquareBoldIcon,
+    StandaloneSquareMinusFillIcon,
+    LabelPairedCircleInfoSmRegularIcon,
+    LabelPairedCircleInfoMdRegularIcon,
+} from "@deriv/quill-icons";
+import { Text } from "../../Typography";
+
+interface CheckboxProps
+    extends Omit<
+        ComponentProps<"input">,
+        "placeholder" | "defaultChecked" | "size"
+    > {
+    indeterminate?: boolean;
+    isLabelPaired?: boolean;
+    infoIconClassName?: string;
+    size?: "sm" | "md";
```

Comment:
I have added `TMediumSizes` you can use it instead.

Reply:
Thank you, applied